Dockerfile

```
@@ -0,0 +1,17 @@
+FROM node:16-alpine as runner
+
+WORKDIR /usr/src/pine
+
+# when switching over to node:17 this v may apply
+# ENV NODE_OPTIONS=--openssl-legacy-provider
+
+COPY . ./
+RUN npm install
+
+
+FROM runner as sut
+CMD /bin/bash -c "npm run mocha"
```

```suggestion
CMD npm run mocha
```
should be fine right?